2001-3000/LIT2246salotz.xml

```diff
@@ -4,9 +4,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
+            <title xml:lang="gez" xml:id="t2">ጸሎት፡ ዘእግዝእትነ፡ ማርያም፡ ዘሰኔ፡ ጎልጎታ፡</title>
+            <title xml:lang="gez" corresp="#t2" type="normalized">Ṣalot za-ʾǝgzǝʾǝtǝna Māryām za-sane Golgotā</title>
+            <title xml:lang="en" corresp="#t2">Prayer of Our Lady Mary in Sane on  Golgotha</title>
```

Comment:
maybe, we add "in the Month of"? Not many know what is "Sane".

Reply:
I've added it, do you think we can keep it as literal translation?

Reply:
It will not be a literal translation. 
A nice descriptive title that accurately reflects the content: "The prayer of Our Lady Mary which she prayed at Golgotha on 21 Sane" (that I personally would prefer)